Makefile

```
@@ -624,3 +624,25 @@ git-submodule-sync: ## Sync and update git submodules.
+
+.PHONY: shell-test-dev
+shell-test-dev: ## Run shell tests inside the Nix dev shell (mirrors CI).
+	@echo "🧪 Running shell tests inside the Nix dev shell..."
+	@DEVENV_ROOT=$(CURDIR) $(NIX_ALLOW_UNFREE) $(NIX_EXEC) develop $(NIX_FLAGS) .# --command $(MAKE) shell-test
+
+.PHONY: shell-check
+shell-check: ## Run ShellCheck on shell scripts.
+	@echo "🔍 Running ShellCheck..."
+	@find . -name '*.sh' -not -path './node_modules/*' -not -path './.git/*' -not -path './result/*' | xargs shellcheck
+
+.PHONY: shell-check-dev
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)